src/TerraDevSite.jsx

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 
 import ApplicationBase from 'terra-application';
-import ApplicationNavigation from 'terra-application-navigation';
+import ApplicationNavigation from 'terra-application/lib/application-navigation';
```

Comment:
We may want to explicitly call out the changes around NavigationPrompt handling that this change introduces.

Reply:
i added more text to the changelog